src/redux/modules/mdb.js

```diff
@@ -159,13 +171,17 @@ const setStatus = (state, action) => {
   const wip     = { ...state.wip };
   const errors  = { ...state.errors };
   const fetched = { ...state.fetched };
-
+  let units     = { errors: {}, wip: {} };
 
   switch (action.type) {
     case FETCH_UNIT:
-      wip.units = { ...wip.units, [action.payload]: true };
+      wip.units     = { ...wip.units, [action.payload]: true };
       fetched.units = { ...fetched.units, [action.payload]: true };
       break;
+    case FETCH_UNITS_BY_IDS:
+      units.wip = action.payload.id?.reduce((acc, id) => ({ ...acc, [id]: true }), {});
+      wip.units = { ...wip.units, ...units.wip };
```

Comment:
This should be a full fetch, right? in that case, in that case also update 

fetched.units = { ...fetched,units, all the new uids that you are going to load... }

see line 179

Reply:
no, I fetch with params, so it can be not full (on this place I need only files)